CONTRIBUTING.md

```diff
@@ -1,60 +1,69 @@
+- If you are unsure about something, don't hesitate to ask the community.
 
-### How do I contribute?
+## 🚨 Contributing best practices
+>  - Please `only work on one` issue at a time.
+>  - If you're unable to continue with an assigned task, inform us promptly. 
+>  - Ensure to `TEST` your feature contributions locally before requesting reviews. 
+>  - Need assistance? Utilize the issue or `help-forum` on [Discord](https://discord.gg/SxH6KUCGH7)
+>  - While Generative AI can be useful, minimize its use for `direct team communication`. We value concise, genuine exchanges over scripted messages.
 
-- Fork the repository and clone it locally
-- Create a new branch and follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines for work undertaken
-- Assign yourself to the issue, if you are working on it (if you are not a member of the organization, please leave a comment on the issue)
-- Make your changes
-- Keep pull requests small and focused, if you have multiple changes, please open multiple PRs
-- Create a pull request back to the upstream repository and follow the [pull request template](.github/pull_request_template.md) guidelines.
-- Wait for a review and address any comments
 
-### Opening PRs
+## How to contribute? 🤷
+---
 
-- As long as you are working on your PR, please mark it as a draft
-- Please make sure that your PR is up-to-date with the latest changes in `main`
-- Fill out the PR template
-- Mention the issue that your PR is addressing (closes: #<id>)
-- Make sure that your PR passes all checks
-- Make sure to test your changes
-- If you have multiple commits in your PR, that solve the same problem, please squash the commits
+Following these steps will ensure that your contributions are well-received, reviewed, and integrated effectively into Komiser's codebase.
 
-### Reviewing PRs
+### Issue assigning 
+1. Assign yourself to the issue, if you are working on it (if you are not a member of the organization, please leave a comment on the issue and we will assign you to it.)
 
-- Be respectful and constructive
-- Assign yourself to the PR
-- Check if all checks are passing
-- Suggest changes instead of simply commenting on found issues
-- If you are unsure about something, ask the author
-- If you are not sure if the changes work, try them out
-- Reach out to other reviewers if you are unsure about something
-- If you are happy with the changes, approve the PR
-- Merge the PR once it has all approvals and the checks are passing
+### Fork and Pull Request Flow 🪜
+
+1. Head over to the [Glasskube GitHub repo](https://github.com/glasskube/glasskube) and "fork it" into your own GitHub account.
+2. Clone your fork to your local machine, using the following command:
+```bash
+git clone git@github.com:USERNAME/FORKED-PROJECT.git
+```
+
+3. Create a new branch based-off **\`main\`** branch:
+```bash
+git checkout main
+git checkout -b fix/XXX-something
```

Comment:
we currently have unwritten _practice_ of using the GitHub username as first part of the branch e.g. `pmig/contributing-guide-improvements` this practice ensures that if multiple people work on the same issue branches will stay unique.